frappe/public/js/frappe/form/footer/base_timeline.js

```diff
@@ -103,7 +103,7 @@ class BaseTimeline {
 			timeline_item.append(
 				`<div class="timeline-load-more">
 					<button class="btn btn-default btn-sm btn-load-more">
-						<span>${item.content}</span>
+						<span>${__(item.content)}</span>
```

Comment:
I think this is not the correct place for translation. At this point, `item.content` can already contain arbitrary HTML that would be impossible to translate.